Sources_v3/StreamChatUI/ChatChannelList/ChatChannelAvatarView.swift

```diff
@@ -40,6 +40,12 @@ open class ChatChannelAvatarView<ExtraData: UIExtraDataTypes>: AvatarView {
             return
         }
 
+        if channel.isDirectMessageChannel,
+            let member = channel.cachedMembers.first(where: { $0.isOnline }),
+            member.isOnline {
+            onlineIndicatorView.availabilityStatus = .online
+        }
+
```

Comment:
private extension Channel { 
  func shouldShowOnlineIdicator(for userId) -> Bool { ... }
}

indicator.isHidden = !channel.shouldShowOnlineIdicator(for: userId)